Dockerfile

```diff
@@ -14,18 +14,19 @@ ARG BASE_OSG_SERIES=3.6
 ARG BASE_YUM_REPO=testing
 ARG TIMESTAMP_TAG
 
-# token auth require HTCondor 8.9.x
+# Grab HTCondor from osg-upcoming-testing, but only that package
 RUN useradd osg \
  && mkdir -p ~osg/.condor \
  && yum -y install \
-        condor \
         osg-wn-client \
         redhat-lsb-core \
         singularity \
         attr \
         git \
         rsyslog rsyslog-gnutls python3-cryptography python3-requests \
         bind-utils \
+ && yum -y install --enablerepo=osg-upcoming-testing \
+        condor \
```

Comment:
Can we wrap this in an `if [[ $BASE_YUM_REPO == "release" ]]`?

Reply:
Oh and a `# FIXME` comment to remove it after today's release. We may bring it back depending on what we decide here https://opensciencegrid.atlassian.net/browse/OSPOOL-17?focusedCommentId=368979

Reply:
> Oh and a # FIXME comment to remove it after today's release. We may bring it back depending on what we decide here https://opensciencegrid.atlassian.net/browse/OSPOOL-17?focusedCommentId=368979

On second thought, don't add the FIXME comment. Let's just go with this solution